packages/json-api/src/Registry.test.ts

```diff
@@ -263,7 +314,12 @@ describe("Registry.parse", () => {
       })
     ).toEqual({
       id: "12",
-      children: [{ id: "123" }, { id: "125" }, { id: "127" }]
+      children: [{ id: "123" }, { id: "125" }, { id: "127" }],
+      included: {
+        "children:123": { id: "123" },
```

Comment:
imho it is better to have resource types as keys

children: [{ id: "123" }, { id: "127" }]